mmcv/runner/base_runner.py

```diff
@@ -330,6 +330,16 @@ def resume(self,
 
         self._epoch = checkpoint['meta']['epoch']
         self._iter = checkpoint['meta']['iter']
+
+        # Recalculate iter when the number of GPUs is different during resume
+        if 'config' in checkpoint['meta']:
+            config = mmcv.Config.fromstring(
+                checkpoint['meta']['config'], file_format='.py')
+            previous_gpu_ids = config.get('gpu_ids', None)
+            if previous_gpu_ids and len(previous_gpu_ids) > 0:
+                self._iter = int(self._iter * len(previous_gpu_ids) /
```

Comment:
Log the information when change self._iter to indicate users that the iteration number is changed due to change of GPU number